Files/Strings/en-US/Resources.resw

```diff
@@ -2067,6 +2067,12 @@
   <data name="PropertyISO" xml:space="preserve">
     <value>ISO speed</value>
   </data>
+  <data name="SettingsPreferencesOpenFoldersNewTab.Header" xml:space="preserve">
+    <value>Open folders in new tab</value>
+  </data>
+  <data name="NavToolbarShowBundlesWidget" xml:space="preserve">
```

Comment:
This resource is a duplicate, and I believe it is what is causing the build error.